### ISSUE TITLE
Mouseclick event does not work on OSC

### ISSUE THREAD

**Issue by Amaranthes** [mpv-player/mpv#2596]
`mouse x y` properly triggers the display of the OSC.
But `mouse x y 0` doesn't work on the OSC, for example triggering this mouse event on the seek bar has no effect.

Same with `keypress MOUSE_BTN0`

**Comment by ChrisK2** [comment]
wat

**Comment by Amaranthes** [comment]
Just an example:

If I put `u mouse 640 610 0` in input.conf, then pressing u should trigger a seek, since it should hit the seekbar of the OSC, given that the video file is 1280x720.

**Comment by ChrisK2** [comment]
Ah okay, you mean input commands ... didn't even know you can use a keybinding to press keys ...

Putting the question why you even want to do this: The OSC only does something when it is visible, so performing a click where an element is _supposed_ to be is not going to work. A "click" is also only recognised when a mouse-down event gets sent followed by a mouse up even (on the same element), maybe the code to fake clicks sends actual "click" events, or only down or up, which the OSC won't recognize.

**Comment by Amaranthes** [comment]
Thanks.
`keydown` actually works.
But I think the OSC should also recognize `keypress`.

> didn't even know you can use a keybinding to press keys ...
> Putting the question why you even want to do this
I suppose it's mandatory if someone writes a GUI or is using mpv in some other application.

**Comment by ChrisK2** [comment]
> I suppose it's mandatory if someone writes a GUI or is using mpv in some other application.

Uh, no. You can do everything the OSC does through the command-interface, in fact the OSC itself is using it to control the player.

**Comment by Amaranthes** [comment]
Oh sorry, I misinterpreted your comment, I actually meant the command interface.
My example with the keybindings was just to demonstrate the issue.

**Comment by wm4** [comment]
I don't care enough about this ridiculous corner case to even investigate this mess again.

**Comment by flaviospagnuolo** [comment]
Hi... I used to embed the MPV player in a Qt desktop app by embedding it through the wId property. After running into a lot of issues, I moved it to the render API (with opengl-cb) and now I'm trying to make the OSC work again. In doing that, now I need to listen to mouse events and send them to the player, but the OSC isn't getting the mouse actions because of this particular issue. Any advice of any other approach would be appreciated.
